.github/actions/e2e-test/action.yml

```diff
@@ -0,0 +1,65 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+name: 'Run E2E Test'
+description: 'Common steps when running an E2E Test'
+inputs:
+  test_class:
+    description: 'The e2e test class name'
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Check License
+      shell: bash
+      run: |
+        echo "::group::Build License-Eye Binary"
+        git clone https://github.com/apache/skywalking-eyes.git
+        git -C skywalking-eyes reset --hard 9bd5feb86b5817aa6072b008f9866a2c3bbc8587
+        make -C skywalking-eyes docker
+        rm -rf skywalking-eyes
+        docker run --rm -v $(pwd):/github/workspace license-eye:latest header check
+        echo "::endgroup::"
+    - name: Build Docker Image
+      shell: bash
+      env:
+        SKIP_TEST: "true"
+      run: |
+        echo "::group::Build Docker Image"
+        make docker && ES_VERSION=es7 TAG=latest-es7 make docker.oap
```

Comment:
Are we always using ES7?

Reply:
> Are we always using ES7?

Not necessary, if we already build a ES6 image, it costs only very little time to build ES7 too, so I wanted to make it simple